README.md

```diff
@@ -67,7 +67,7 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
  - [GraphQL Inspector Action](https://github.com/kamilkisiela/graphql-inspector)
  - [Snyk CLI Test Action](https://github.com/clarkio/snyk-cli-action)
  - [PowerShell static analysis with PSScriptAnalyzer](https://github.com/devblackops/github-action-psscriptanalyzer)
- - [Run web performance audits using Sitespeed.io](https://github.com/sitespeedio/sitespeed.io/tree/master/docker/github-actions)
+ - [Run web performance audits using Sitespeed.io](https://github.com/sitespeedio/sitespeed.io/tree/ad1716bb9227b85ad80cbca65284c73b9a822ec5/docker/github-action)
```

Comment:
@saracope is there a reason that you have included a commit id in this URL?  Doesn't this URL make sense:

https://github.com/sitespeedio/sitespeed.io/tree/master/docker/github-action

Reply:
Oh, weird, I didn't notice it added that in. Updated, thanks!